src/GraphQL/Schemas/Primary/Mutations/MarkAndListPendingTransactionsMutation.php

```diff
@@ -77,7 +77,9 @@ function (Builder $query) use ($args) {
                     return $query->whereIn('wallet_public_key', $publicKeys);
                 },
                 function (Builder $query) {
-                    return $query->whereIn('wallet_public_key', Account::managedPublicKeys());
+                    return $query
```

Comment:
I think query will look like this and can cause unexpected result
```select * from transactions where state = 'PENDING' and wallet_public_key in('public keys'...) OR wallet_public_key = null```

the correct query should be 
```select * from transactions where state = 'PENDING' and (wallet_public_key in('public keys'...) OR wallet_public_key is null)```


```suggestion
            return $query->where(function($query) {
                   ->whereIn('wallet_public_key', Account::managedPublicKeys())
                   ->orWhereNull('wallet_public_key');
            });
```
FYI, i haven't tested this but code should be something like this

Reply:
Anyway i think query is still wrong, we shouldn't do `wallet_public_key = null` for null comparison instead it should be `wallet_public_key is null`

https://dev.mysql.com/doc/refman/8.0/en/working-with-null.html

Reply:
Well we are using laravel:
```
        // If the value is "null", we will just assume the developer wants to add a
        // where null clause to the query. So, we will allow a short-cut here to
        // that method for convenience so the developer doesn't have to check.
        if (is_null($value)) {
            return $this->whereNull($column, $boolean, $operator !== '=');
        }
```

Reply:
Good to know